language-support/ts/daml-json-types/BUILD.bazel

```diff
@@ -27,3 +31,17 @@ pkg_npm(
     visibility = ["//visibility:public"],
     deps = ["daml-json-types"],
 )
+
+jest_test(
+    name = "test",
+    srcs = glob(["**/*.ts"]),
```

Comment:
Should this only `glob` `*.test.ts` and then add `":daml-json-types"` to `deps`?

Reply:
I had already tried that and couldn't get it to work. 😞